src/lightkurve/config/__init__.py

```diff
@@ -0,0 +1,32 @@
+import astropy.config as astropyconfig
+
+
+ROOTNAME = 'lightkurve'
+
+
+class ConfigNamespace(astropyconfig.ConfigNamespace):
+    rootname = ROOTNAME
+
+
+class ConfigItem(astropyconfig.ConfigItem):
+    rootname = ROOTNAME
+
+
+def get_config_dir():
+    """
+    Determines the package configuration directory name and creates the
+    directory if it doesn't exist.
+
+    This directory is typically ``$HOME/.lightkurve/config``, but if the
+    XDG_CONFIG_HOME environment variable is set and the
+    ``$XDG_CONFIG_HOME/lightkurve`` directory exists, it will be that directory.
+    If neither exists, the former will be created and symlinked to the latter.
```

Comment:
Thank you for including this detailed explanation on the behavior of XDG_CONFIG_HOME.  I had no idea!

Reply:
Kudos to Astropy contributors: The docstring is effectively a copy of the existing one at:
https://docs.astropy.org/en/stable/api/astropy.config.get_config_dir.html#astropy.config.get_config_dir